resources/active_directory.rb

```diff
@@ -33,11 +33,13 @@
   include SensuCookbook::Helpers
 end
 
-property :ad_servers, Array, required: true
+property :auth_servers, Array, required: true
 property :groups_prefix, String
 property :username_prefix, String
 property :resource_type, String, default: 'ad'
-alias_method :servers, :ad_servers
+# maintain backward compat with released cookbook versions for now
```

Comment:
💯 we will look to remove those deprecations next major release.

Reply:
updated changelog